unifi/setting_mgmt.go

```diff
@@ -9,5 +9,6 @@ func (c *Client) GetSettingMgmt(ctx context.Context, site string) (*SettingMgmt,
 }
 
 func (c *Client) UpdateSettingMgmt(ctx context.Context, site string, d *SettingMgmt) (*SettingMgmt, error) {
+	d.Key = "mgmt"
```

Comment:
We should really update the generation for settings to do this automatically probably. I'll open a new issue for that.